pkg/kv/kvserver/concurrency/concurrency_manager.go

```diff
@@ -182,33 +185,46 @@ func (m *managerImpl) SequenceReq(
 	ctx context.Context, prev *Guard, req Request, evalKind RequestEvalKind,
 ) (*Guard, Response, *Error) {
 	var g *Guard
+	var branch int
 	if prev == nil {
 		switch evalKind {
 		case PessimisticEval:
+			branch = 1
 			log.Event(ctx, "sequencing request")
 		case OptimisticEval:
+			branch = 2
 			log.Event(ctx, "optimistically sequencing request")
 		case PessimisticAfterFailedOptimisticEval:
+			branch = 3
```

Comment:
these branches will not fire given the panic underneath, right?

maybe you want a `defer` that wraps the branch number around the panic object?

Reply:
No, they won't fire, so this is intentionally dead code. I removed it.